### ISSUE TITLE
[FEAT] ujust rebase-secureblue

### ISSUE THREAD

**Issue by mintpilo** [secureblue/secureblue#692]
**Describe the benefit this feature would provide to the project**
Allow users to more easily rebase to alternative secureblue image's such as kinonite, nvidia-open, (as opposed to nvidia) etc. once they've already installed secureblue.

**Describe the solution you'd like**
ujust script potentially based on `install_secureblue.sh` (or directly calling it) that would streamline rebasing to other secureblue images once they're already inside a secureblue image. 

**Declaration**
- [x] I declare that I have read the README and my feature request is in-scope.

**Comment by ShadowSlayer1441** [comment]
I mean this would literally just require creating an rebase.just files in files/justfiles/, and prefacing the current script with "rebase-secureBlue-Desktop:" Or am I missing something, ujust runs as sh by default, so it should just work. Maybe reword some stuff.
